src/importlinter/domain/fields.py

```diff
@@ -181,25 +198,11 @@ def parse(self, raw_data: Union[str, List]) -> ImportExpression:
         if not (importer and imported):
             raise ValidationError('Must be in the form "package.importer -> package.imported".')
 
-        self._validate_wildcard(importer)
-        self._validate_wildcard(imported)
+        _validate_wildcard(importer)
```

Comment:
Not a blocker, but if we introduce the concept of `ModuleExpression`, it might be nice to use them within `ImportExpression` objects, which could become a pair of ModuleExpressions.